readme.md

```diff
@@ -0,0 +1,42 @@
+# Maintenance Mode
+
+[![Build Status](https://api.travis-ci.com/Automattic/maintenance-mode.svg?branch=master)](https://api.travis-ci.com/Automattic/maintenance-mode)
+[![GPLv2 License](https://img.shields.io/github/license/Automattic/maintenance-mode.svg)](https://www.gnu.org/licenses/gpl.html)
```

Comment:
Use a version-specific links, since this gpl.html is for V3.

https://www.gnu.org/licenses/old-licenses/gpl-2.0.html is for V2.

Reply:
Good catch. I missed that one.